### PR TITLE
Ops tools2 save ca cert

### LINKED ISSUE #1719: Ops Tools - Copy CA certificate chain to the environment base directory
If a CA certificate chain is provided, we need to store it so that we don't ask for it again on reconnect/Edit filters/refresh.

This file should be stored on the environment base directory (one up from nodes).

### PR DESCRIPTION
**Pull request by lquintai**
Save CA certificate chain file in the environment base directory when provided by the user.

closes #1719

### CODE REVIEW

File: packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts [hunk 1]
```diff
@@ -221,11 +235,10 @@ export async function importNodesToEnvironment(environmentRegistryEntry: FabricE
 
         const dirPath: string = await vscode.workspace.getConfiguration().get(SettingConfigurations.EXTENSION_DIRECTORY) as string;
         const homeExtDir: string = FileSystemUtil.getDirPath(dirPath);
-        const environmentPath: string = path.join(homeExtDir, 'environments', environmentRegistryEntry.name, 'nodes');
+        const environmentNodesPath: string = path.join(homeExtDir, 'environments', environmentRegistryEntry.name, 'nodes');
```
Comment (cazfletch) [inline packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts]: you don't need all this you can use your variable you have at the top for the `environmentBaseDir`

File: packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts [hunk 2]
```diff
@@ -251,7 +264,7 @@ export async function importNodesToEnvironment(environmentRegistryEntry: FabricE
         const newEnvironment: FabricEnvironment = new FabricEnvironment(environmentRegistryEntry.name);
         const newNodes: FabricNode[] = await newEnvironment.getNodes();
         if (newNodes.length === 0) {
-            await fs.remove(environmentPath);
+            await fs.remove(environmentNodesPath);
```
Comment (cazfletch) [inline packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts]: in the case of a non ops tools one this needs to remove the environment

File: packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts (hunk 1 above)
Comment (lquintai) [reply packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts]: True, will refactor.

File: packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts (hunk 2 above)
Comment (lquintai) [reply packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts]: Indeed, the behavior is now different from what is currently expected, I'll change that.

File: packages/blockchain-extension/extension/commands/addEnvironmentCommand.ts [hunk 3]
```diff
@@ -62,6 +61,9 @@ export async function addEnvironment(): Promise<void> {
         }
         Reporter.instance().sendTelemetryEvent('addEnvironmentCommand');
     } catch (error) {
+        if (fabricEnvironmentEntry.name) {
```
Comment (cazfletch) [inline packages/blockchain-extension/extension/commands/addEnvironmentCommand.ts]: You don't need this bit as the delete will ignore the error if it doesn't exist

**Pull request merged**